addons/website/views/website_templates.xml

```diff
@@ -2378,7 +2378,7 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
         <t t-set="search_placeholder">Search...</t>
         <input type="search" name="search" t-att-class="'search-query form-control oe_search_box %s' % _input_classes" t-att-placeholder="placeholder if placeholder else search_placeholder" t-att-value="search"/>
         <button type="submit" t-att-class="'btn oe_search_button %s' % (_submit_classes or 'btn-primary')" aria-label="Search" title="Search">
-            <i class="fa fa-search"/>
+            <i class="oi oi-search"/>
```

Comment:
Did not you remove the icons.scss file from frontend following my first review?

Reply:
Yes. 
`icons.scss` doesn't add any new icon, it defines utility-classes to handle them.
The 13 `oi` icons are already included in the frontend but should be used for not editable elements only.